source/cloud-security/amazon/services/supported-services/security-hub.rst

```diff
@@ -0,0 +1,180 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: Learn how to configure Amazon Security Hub findings and insights fetching.
+
+AWS Security Hub
+================
+
+.. versionadded:: 4.9.0
+
+`AWS Security Hub <https://aws.amazon.com/security-hub/>`_ is a cloud security posture management (CSPM) service that automates security best practice checks, aggregates security alerts into a determined format, and helps the user understand the overall security posture across all of the AWS accounts.
+
+Security Hub helps users assess their compliance against security best practices as follows:
+
+-  Runs checks against security controls.
+-  Generates control `findings <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings.html>`__.
+-  Groups related findings into collections called `insights <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-insights.html>`__.
+
+Wazuh integrates with `Amazon SQS <https://aws.amazon.com/sqs>`_ and `EventBridge <https://aws.amazon.com/eventbridge>`_ to centralize Security Hub findings and insights in a single place. To set up the integration, you need to:
+
+#. Configure AWS:
```

Comment:
Since they are ordered steps we better use an ordered list:

```
#. Configure AWS. This involves the following.

   #. Enabling Amazon Security Hub.
   #. Integrating Security Hub with EventBridge.
   #. Enabling an Amazon SQS queue.
   #. Enabling an Amazon S3 bucket including Event notifications. The bucket sends notifications to the queue for every Security Hub object creation event.
#. Set up the Wazuh integration for Amazon Security Hub.
```